core/tests/puppeteer-acceptance-tests/specs/logged-out-user/use-keyboard-shortcuts-to-navigate-and-shift-focus.spec.ts

```diff
@@ -0,0 +1,133 @@
+import {ExplorationEditor} from '../../utilities/user/exploration-editor';
+
+const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
+const CONCEPT_CARD_CONTENT_EN = 'Numbers can be positive, negative, or zero.';
+enum INTERACTION_TYPES {
+  CONTINUE_BUTTON = 'Continue Button',
+  NUMERIC_INPUT = 'Number Input',
+  END_EXPLORATION = 'End Exploration',
+}
+enum CARD_NAME {
+  INTRODUCTION = 'Introduction',
+  TEST_QUESTION = 'Test Question',
+  CONCEPT_CARD = 'Concept Card',
+  FINAL_CARD = 'Final Card',
+}
+
+describe('Logged-out User', function () {
+  let explorationEditor: ExplorationEditor;
+  let loggedOutUser: LoggedOutUser;
+
+  beforeAll(async function () {
+    explorationEditor = await UserFactory.createNewUser(
+      'explorationEditor',
+      'exploration_editor@example.com'
+    );
+
+    await explorationEditor.navigateToCreatorDashboardPage();
+    await explorationEditor.navigateToExplorationEditorPage();
+    await explorationEditor.dismissWelcomeModal();
+    await explorationEditor.updateCardContent(
+      'We wile be learning numbers today.'
+    );
+    await explorationEditor.addInteraction(INTERACTION_TYPES.CONTINUE_BUTTON);
+
+    // Add a new card with a question.
+    await explorationEditor.viewOppiaResponses();
+    await explorationEditor.directLearnersToNewCard(CARD_NAME.CONCEPT_CARD);
+    await explorationEditor.saveExplorationDraft();
+
+    // Navigate to the new card and add concept content.
+    await explorationEditor.navigateToCard(CARD_NAME.CONCEPT_CARD);
+    await explorationEditor.updateCardContent(CONCEPT_CARD_CONTENT_EN);
+    await explorationEditor.addInteraction(INTERACTION_TYPES.CONTINUE_BUTTON);
+    await explorationEditor.editDefaultResponseFeedback(
+      undefined,
+      CARD_NAME.FINAL_CARD
+    );
+    await explorationEditor.saveExplorationDraft();
+
+    // Navigate to the final card and update its content.
+    await explorationEditor.navigateToCard(CARD_NAME.FINAL_CARD);
+    await explorationEditor.updateCardContent(
+      'We have practiced positive numbers.'
+    );
+    await explorationEditor.addInteraction(INTERACTION_TYPES.END_EXPLORATION);
+
+    // Navigate back to the introduction card and save the draft.
+    await explorationEditor.navigateToCard(CARD_NAME.INTRODUCTION);
+    await explorationEditor.saveExplorationDraft();
+
+    await explorationEditor.publishExplorationWithMetadata(
+      'Positive Numbers',
+      'Learn positive numbers.',
+      'Algebra'
+    );
+
+    loggedOutUser = await UserFactory.createLoggedOutUser();
+  }, DEFAULT_SPEC_TIMEOUT_MSECS);
+
+  it(
+    'should be able to navigate the site using keyboard shortcuts, check focus, and interact with the exploration player.',
+    async function () {
+      // Navigate to the Get Started page using the ‘Ctrl+6’ shortcut.
+      await loggedOutUser.simulateKeyboardShortcut('Ctrl+6');
+      await loggedOutUser.expectToBeOnPage('get-started');
```

Comment:
I don't think this is a good way to check. You should check the full URL. In expectToBeOnPage, maybe check the full URL by checking if it's 
`localhost:8181/ + get-started`

Reply:
Yes, maybe we can use the URL too.
However, would it be better to just pass the name of the page? because some of the login pages will have return url, which can make the many login urls and lil complex, which may not be needed. 

Also, to make the present implementation better, I have removed the need to include '-' in the page name so, a name - 'community library' would be good enough to check if the user is onto that page or not. 

Would this be fine?

Reply:
That's a good point. Let's just leave it like this right now. I agree '-' should be removed.